terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/components.tf

```diff
@@ -106,12 +106,11 @@ module "monitoring" {
   oidc_components_client_secret = data.terraform_remote_state.cluster.outputs.oidc_components_client_secret
   oidc_issuer_url               = data.terraform_remote_state.cluster.outputs.oidc_issuer_url
   enable_thanos_sidecar         = terraform.workspace == local.live_workspace ? true : false
+  enable_large_nodesgroup       = terraform.workspace == local.live_workspace ? true : false
```

Comment:
@mogaal , this will set `enable_large_nodesgroup=true` for manager as the live_workspace is set to manager in here. 
https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/main/terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/main.tf#L69-L72

Reply:
You are 100% right, fixing it now. Thanks for that.